evap/staff/templates/staff_infotexts.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'staff_base.html' %}
+
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item">{% trans 'Infotexts' %}</li>
+{% endblock %}
+
+{% block content %}
+    {{ block.super }}
+
+    <form method="POST" class="form-horizontal" id="infotext-formset">
+        {% csrf_token %}
+        {{ formset.management_form }}
+        <div class="card mb-3">
+            <div class="card-body">
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th>{% trans 'Title/Content (German)' %}</th>
```

Comment:
I wonder if it would be better to have headings "Title" / "Content" at the fields themselves, not on top - not sure if it looks better though. I feel like everything looks a bit empty right now

Reply:
The page would actually better fit the overall layout if the three sections would be placed inside cards with the page names in the cards' headers. The "Title/Content (German)" headings would then be placed on top of the fields within each card.

Reply:
you can check out two first versions: `card layout` and `css grid instead of table`